### PR TITLE
edit_destroy

### PR DESCRIPTION
**Pull request by phannh58**

### CODE REVIEW

**Comment by k-i-k-ichi** [comment]
ok

File: app/views/admin/categories/_category.html.erb [hunk 1]
```diff
@@ -7,13 +7,45 @@
...
       <div class="date_time"><%= localize_time category.created_at %></div>
       <br>
-      <div class="btn-group">
-        <%= link_to t("views.categories.edit"),
-          "#", class: "btn btn-success" %>
-        <%= link_to t("views.categories.delete"), "#",
-          method: :delete, data: {confirm: t("messages.notice")},
-          class: "btn btn-danger" %>
+      <div class="btn-group" role="group" aria-label="...">
+        <button type="button" class="btn btn-warning" data-toggle="modal" data-target="#myupdatecategory_<%= category.id %>">
```
Comment (dogiadat) [inline app/views/admin/categories/_category.html.erb]: dài nha, cắt bớt đi

File: app/views/admin/categories/_category.html.erb [hunk 2]
```diff
@@ -7,13 +7,45 @@
...
+          <%= t "views.categories.edit" %></button>
+        <%= link_to t("views.categories.delete"), [:admin, category],
+          method: :delete, class: "btn btn-danger", remote: true %>
       </div>
     </div>
   </div>
+  <div class="modal fade" id="myupdatecategory_<%= category.id %>" tabindex="-1"
+  role="dialog"  aria-labelledby="myModalLabel" aria-hidden="true">
```
Comment (dogiadat) [inline app/views/admin/categories/_category.html.erb]: indent

File: app/views/admin/categories/_category.html.erb [hunk 3]
```diff
@@ -7,13 +7,45 @@
...
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```
Comment (dogiadat) [inline app/views/admin/categories/_category.html.erb]: dài

File: app/views/admin/categories/_category.html.erb [hunk 4]
```diff
@@ -7,13 +7,47 @@
...
+        <button type="button" class="btn btn-warning"
+          data-toggle="modal" data-target="#myupdatecategory_<%= category.id %>">
+          <%= t "views.categories.edit" %></button>
+        <%= link_to t("views.categories.delete"), [:admin, category],
+          method: :delete, class: "btn btn-danger", remote: true %>
       </div>
     </div>
   </div>
+  <div class="modal fade" id="myupdatecategory_<%= category.id %>" tabindex="-1"
+  role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```
Comment (dogiadat) [inline app/views/admin/categories/_category.html.erb]: indent

**Comment by dogiadat** [comment]
ok

File: app/views/admin/categories/_category.html.erb [hunk 5]
```diff
@@ -7,12 +7,46 @@
...
+    role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal"
+            aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="myModalLabel">
+            <%= t "views.categories.edit" %></h4>
+        </div>
+        <%= form_for [:admin, category], :method => :put, remote: true do |f| %>
```
Comment (dieunb) [inline app/views/admin/categories/_category.html.erb]: `method: :put`

**Pull request merged**